From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Dec 2021 11:23:37 +0000
Subject: Hard-code tput to be found at /usr/bin/tput

Otherwise, if ksh is built on a merged-/usr system, we might find it
at /bin/tput, which is equivalent to /usr/bin/tput on a merged-/usr
system, but does not exist if the resulting package is used on a
traditional split-/usr system.

Similarly, if /usr/local/bin/tput exists on the build system, it will
probably not exist on the system where the resulting package is to be
used.

In Autotools, this would usually be done by setting TPUT=/usr/bin/tput
at configure time, but this package uses its own vaguely similar setup
instead of an Autotools configure script and does not appear to offer
a corresponding mechanism to override the auto-detected path, so we'll
have to hard-code this as a Debian-specific change.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: not-needed, Debian-specific
---
 src/cmd/ksh93/features/cmds | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cmd/ksh93/features/cmds b/src/cmd/ksh93/features/cmds
index 87d211c..171752e 100644
--- a/src/cmd/ksh93/features/cmds
+++ b/src/cmd/ksh93/features/cmds
@@ -4,7 +4,7 @@ pth	ed fail{
 	echo '#define _pth_ed	"ed" /* ed not found on standard PATH */'
 }end
 
-pth	tput
+pth	tput	/usr/bin
 
 tput_terminfo note{ does tput support terminfo codes }end run{
 	TERM=ansi
